tasks/cert_sync_detection.yml

```diff
@@ -22,7 +22,7 @@
   shell:
     cmd: |
       set -o pipefail
-      ls -1 *.csr | { grep -v server.csr || true; } | sort | cut -f1 -d'.' > {{ tempfile_existing.path }}
+      ls -1 *.csr | { grep -v server.csr || true; } | cut -f1 -d'.' | sort > {{ tempfile_existing.path }}
```

Comment:
Do you have an example of how this broke?

I see your commit mentioning non-lexicographic sorting, which is kind of weird given that it's supposed to be taking the first field before the `.`, which afaik sort did by default.

Reply:
It will affects you when you have user list sorted like this:
```
auser
buser
buser-mobile
cuser
```

File sorting will be different (which comm util cannot handle)

example:
```
~ $ touch auser.csr buser.csr buser-mobile.csr cuser.csr
~ $ ls -1 *.csr | sort | cut -f1 -d'.'
auser
buser-mobile
buser
cuser
~ $ ls -1 *.csr | cut -f1 -d'.' | sort
auser
buser
buser-mobile
cuser
```